### ISSUE TITLE
[Docs] Adjust theming to ensure a 4.5:1 contrast ratio for all small (<24px) text and a 3:1 contrast ratio for all large text and graphics.

### ISSUE THREAD

**Issue by roraback** [gravitational/docs#205]
Currently, all text colors in the docs except black and dark purple fail [WCAG AA standards](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/visual-audio-contrast-contrast.html) of
* ≥4.5:1 for small text
* ≥3:1 for large text and graphics.

Some examples:

![Image](https://user-images.githubusercontent.com/1095531/205744397-8daa1930-297e-4991-b85f-9ed9dab4bf52.png)

-

![Image](https://user-images.githubusercontent.com/1095531/205744860-4a4e6799-6502-4d56-b548-e77acbe16d47.png)

-

![Image](https://user-images.githubusercontent.com/1095531/205745250-5d2112d9-30be-4f78-8a14-2cfb76122cd0.png)

-

![Image](https://user-images.githubusercontent.com/1095531/205746001-6bc0e3aa-844a-4ffa-ac2d-83732af7d504.png)

**Comment by roraback** [comment]
This should be taken care of with bigger redesign work.
